releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -74,59 +104,24 @@ ARGS=(
 )
 ```
 
-- [ ] Stellar → EVM GMP call
+### Stellar → EVM GMP call
 
 ```bash
-node stellar/gmp.js send avalanche 0x0FCb262571be50815627C16Eca1f5F3D342FF5a5 0x1234 CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC 1
-
-axelard tx wasm execute $GATEWAY '{"verify_messages":
-      [{"cc_id":{
-        "source_chain":"'"$CHAIN"'",
-        "message_id":"0x7b903beabbf200f7408c8c46cc4e5281396fff68f718515984ad6481f563be2b-0"},
-        "destination_chain":"avalanche",
-        "destination_address":"0xba76c6980428A0b10CFC5d8ccb61949677A61233","source_address":"GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG","payload_hash":"56570de287d73cd1cb6092bb8fdee6173974955fdef345ae579ee9f475ea7432"}]
-    }' $ARGS
-
-axelard tx wasm execute $GATEWAY '{"route_messages":
-      [{"cc_id":{
-        "source_chain":"'"$CHAIN"'",
-        "message_id":"0x7b903beabbf200f7408c8c46cc4e5281396fff68f718515984ad6481f563be2b-0"},
-        "destination_chain":"avalanche",
-        "destination_address":"0xba76c6980428A0b10CFC5d8ccb61949677A61233","source_address":"GAIA45FE43FHCKPK5A2RYBDTYG3D3TT3F3KHFWGWX53TH7FIMBK6LRSJ","payload_hash":"56570de287d73cd1cb6092bb8fdee6173974955fdef345ae579ee9f475ea7432"}]
-    }' $ARGS
+node stellar/gmp.js send avalanche [destination-address] 0x1234 CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC 1
 
```

Comment:
Add a step that just says something like Route GMP call via Amplifier. Can link to the example amplifier GMP doc we have